app/controllers/communities_controller.rb

```diff
@@ -54,6 +58,19 @@ def destroy
     redirect_to communities_path
   end
 
+  def mod
+    # @community = Community.find(params[:id])
+    # @community.id
+    @banneduser = BannedUser.new
+    @username = Account.pluck(:username).sort
```

Comment:
No magesh  there is no possiblity that there is no user because this functionality is done in moderator search field so atleast there will be owner of the community who is also a user so it will not throw error